releasenotes/notes/bugfix-fipsisntall-script-during-upgrade-73529cb19be22109.yaml

```diff
@@ -0,0 +1,12 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+fixes:
+  - |
+    (datadog-fips-agent) Bugfix for the post install script to always rebuild the
+    fipsmodule.cnf in case the embedded OpenSSL has been updated.
```

Comment:
```suggestion
    (datadog-fips-agent)  Ensure the post-install script always rebuilds fipsmodule.cnf when the embedded OpenSSL is updated.
```

Reply:
fixed in https://github.com/DataDog/datadog-agent/pull/36808/commits/b39a45c0735d3d4427696787a47e5aae7ad6a7c5